static/css/main.css

```diff
@@ -223,424 +225,422 @@ body {
-  background: white;
-  border: 0;
-  border-radius: 8px;
-  float: right;
+    text-align: center;
+    background: #ababab;
+    margin: 10px auto;
+}
+
+div.tooltip {
+    color: #444;
+    position: absolute;
+    text-align: left;
+    padding: 4px;
+    font-size: 14px;
+    line-height: 15px;
+    background: white;
+    border: 0;
+    border-radius: 8px;
+    pointer-events: none;
 }
 
 .alarms {
-  display: none;
+    display: none;
 }
 
 .loading .focus-range {
-  display: none;
+    display: none;
 }
 
 .focus-range {
-  list-style: none;
-  margin: 4px;
-  padding: 0;
-  color: #808080;
-  width: 250px;
-  text-align: center;
+    list-style: none;
+    margin: 4px;
+    padding: 0;
+    color: #808080;
+    width: 250px;
+    text-align: center;
 }
 
 .focus-range li {
-  display: inline-block;
-  font-size: 18px;
-  white-space: nowrap;
-  border-radius: 5px;
-  border: 2px solid #000;
-  cursor: pointer;
-}
-
-.focus-range .selected {
-  border-color: #808080;
-  color: #000;
-  background: #808080;
+    display: inline-block;
+    font-size: 18px;
+    white-space: nowrap;
+    border-radius: 5px;
+    border: 2px solid #000;
+    cursor: pointer;
 }
 
-.pill.hidden {
-  display: none;
+.focus-range li.selected {
+    border-color: #808080;
+    color: #000;
+    background: #808080;
 }
 
 @media (max-width: 800px) {
-  .bgStatus {
-    width: 300px;
-  }
+    .bgStatus {
+        width: 300px;
+    }
 
-  .bgButton {
-    width: auto;
-  }
+    .bgButton {
+        width: auto;
+    }
 
-  .bgStatus .currentBG {
-    font-size: 80px;
-    line-height: 80px;
-  }
+    .bgStatus .currentBG {
+        font-size: 80px;
+        line-height: 80px;
+    }
 
-  .bgStatus .currentBG.bg-limit, .bgStatus .currentBG.icon-hourglass {
-    font-size: 70px;
-  }
+    .bgStatus .currentBG.bg-limit, .bgStatus .currentBG.icon-hourglass {
+      font-size: 70px;
+    }
 
-  .bgStatus .pill.direction {
-    font-size: 70px;
-    line-height: 70px;
-  }
+    .bgStatus .currentDirection {
+        font-size: 70px;
+        line-height: 70px;
+    }
 
-  .bgStatus .majorPills {
-    font-size: 20px;
-    line-height: 24px;
-  }
+    .bgStatus .currentDetails {
+        font-size: 20px;
+    }
 
-  .bgStatus .minorPills {
-    font-size: 16px;
-    line-height: 18px;
-  }
+    .time {
+        font-size: 70px;
+        line-height: 60px;
+        padding-top: 8px;
+    }
 
-  .status {
-    font-size: 70px;
-    padding-top: 8px;
-  }
+    .timeOther {
+        font-size: 15px;
+        line-height: 40px;
+    }
 
-  .statusPills {
-    font-size: 15px;
-    line-height: 17px;
-  }
+    #uploaderBattery label {
+        font-size: 15px !important;
+    }
 
-  .focus-range {
-    margin: 0;
-  }
+    .focus-range {
+        margin: 0;
+    }
 
-  .focus-range li {
-    font-size: 14px;
-  }
+    .focus-range li {
+        font-size: 14px;
+    }
 
-  #silenceBtn a {
-    font-size: 30px;
-  }
-
-  #chartContainer {
-    font-size: 14px;
-  }
-}
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The universal selector (*) is known to be slow.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/969293254/issues/source?bid=2452374&fileBranchId=2861014#l373)